dev/docker/ballista.dockerfile

```
@@ -49,7 +49,9 @@ ARG PROTOC_VERSION=21.4
 RUN mkdir /tmp/protoc
 WORKDIR /tmp/protoc
 
-RUN export PROTO_ZIP="protoc-${PROTOC_VERSION}-linux-x86_64.zip" && \
+RUN CPU_ARCH=`uname -m` && \
+  if [ $CPU_ARCH=="aarch64" ]; then CPU_ARCH="aarch_64"; fi && \
```

Adding double quotes here seems to be necessary to make this work both on Ubuntu AMD and Mac M1.

```suggestion
  if [ "$CPU_ARCH"=="aarch64" ]; then CPU_ARCH="aarch_64"; fi && \
```

---

Actually, I am struggling to make this work in both envs. Ubuntu seems to want spaces around the `==`.

---

Thanks for the suggestions and testing